sov-modules/sov-modules-api/src/lib.rs

```diff
@@ -207,7 +207,5 @@ pub trait ModuleInfo {
 
     fn new() -> Self;
 
-    // Returns an address for the module.
-    // TODO: https://github.com/Sovereign-Labs/sovereign/issues/136
```

Comment:
@bkolad please verify that this removal is correct

Reply:
Yes